src/components/ShoppingList/ShoppingList.js

```diff
@@ -0,0 +1,35 @@
+import { db } from '../../lib/firebase.js';
+import { useCollection } from 'react-firebase-hooks/firestore';
+import ShoppingListItem from '../ShoppingListItem/ShoppingListItem.js';
+
+function ShoppingList({ listId }) {
+  const [listItems, loading, error] = useCollection(
+    db.collection(`lists/${listId}/items`).orderBy('purchaseInterval', 'asc'),
+  );
+
+  return (
+    <div className="shopping-list">
+      {loading && (
+        <div className="shopping-list__notice notice notice_type_loading">
+          Loading...
+        </div>
+      )}
+
+      {error && (
+        <div className="shopping-list__notice notice notice_type_error">
+          Error
+        </div>
+      )}
+
+      {!loading && listItems && (
+        <ul className="shopping-list__list">
```

Comment:
I know. But trying to stick using BEM conventions and component names for 'block' css classes, this seemed the best BEMy name for a class to style the ul inside the ShoppingList component.